rules.yml

```diff
@@ -1500,6 +1500,111 @@ rules:
           name 'example'
           version '1.0.0'
           source_url 'https://github.com/chef-cookbooks/something' if respond_to?(:source_url)
+  - code: FC066
+    name: Ensure chef_version is set in metadata
+    tags:
+      - metadata
+    summary: |
+      This warning is shown if a cookbook does not include the chef_version property
+      in its metadata file. chef_version is used to clearly specify chef version 
+      compatibility to users.
+    examples:
+      - title: Metadata without chef_version
+        code: |
+          name 'example'
+          version '1.0.0'
+      - title: Modified version with chef_version
+        code: |
+          name 'example'
+          version '1.0.0'
+          chef_version '>= 12.5' if respond_to?(:chef_version)
+  - code: FC067
+    name: Ensure at least one platform supported in metadata
+    tags:
+      - metadata
+      - supermarket
+    summary: |
+      This warning is shown if a cookbook does not include any supports properties
+      in its metadata file. supports is used to clearly specify platform support
+      to users.
```

Comment:
We should mention that this is used to add platform badges to Supermarket and allows the user to search by supported platforms.